Portfolio/index.css

```diff
@@ -1 +1,67 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;
+    padding: 10px;
+  
+}
+nav ul{
+    padding: 0px;
+    margin: 0px;
+}
+nav li{
+    list-style-type: none;
+}
+.button{
+    border-radius: 20px;
+    padding: 10px;
+    border-color: #515050;
+}
+img{
+    border-radius: 50%;
+}
+form{
+    border: 2px solid #ffffff66;
+    background-color: #ffffff66;
+    display: inline-block;
+    padding: 5px 40px;
+    border-radius: 15px;
+}
+form h4{
+    background-color: #ffffff66;
+
+}
+form input{
+    width: 400px;
+    padding: 5px 20px;
+    background-color: white;
+}
+form textarea{
+    width: 420px;
+    height: 50px;
+    background-color: white;
+    padding: 5px;
+}
+form button{
+    border-radius: 10px;
+    color: white;
+    padding: 5px 15px;
+    background-color: #444444;
+    position: relative;
+    left: 180px;
+    margin: 10px;
+}
+.cont2{
+    background-color: #444444;
+    padding: 10px;

```

Comment:
Consider using a more consistent approach for positioning elements instead of using 'left' and 'position' properties together